src/Api.php

```diff
@@ -67,6 +76,8 @@ public function __construct(ConfigFactoryInterface $config_factory, $host, Clien
     $this->httpClient = $http_client;
     $this->logger = $logger;
     $this->state = $state;
+    $this->_webhook = $webhook;
+    $this->_base_url = \Drupal::request()->getHost();
```

Comment:
This should be injected rather than using `\Drupal::` since we are in a service.